src/utils/DateUtils.ts

```diff
@@ -331,7 +331,7 @@ export class DateUtils {
     }
 
     if (options.useLongDateFormat) {
-      return `${dateOnly.format('dddd')} ${dateOnly.format('LL')} ${dateOnly.format('YYYY')}`;
+      return `${dateOnly.format('dddd')}, ${dateOnly.format('LL')}, ${dateOnly.format('YYYY')}`;
```

Comment:
As explained during our scrum, this will probably not be valid for different languages.

Could we leverage something in globalize js to get a valid "long date format" ?

Reply:
None of the formats in Globalize contain commas. On the other hand, those in momentjs do. Since we overwrite the moment values with those of globalize, I needed to store the moment values in a map to be able to retrieve them later on.